osu.Game/Online/OnlineViewContainer.cs

```diff
@@ -21,7 +21,7 @@ public abstract class OnlineViewContainer : Container, IOnlineComponent
 
         protected const double TRANSFORM_TIME = 300.0;
 
-        internal readonly Container TransformationTarget;
+        private readonly Container viewTarget;
         protected override Container<Drawable> Content { get; }
```

Comment:
why not extract the backing-field of this property and using it as the "view target", adding a second layer seems redundant.
```csharp
private readonly Container content;
protected override Container<Drawable> Content => content;
```